control-plane/roles/monitoring/templates/alertloggerStatefulSet.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Is there a reason why this is a stateful set and not a deployment? As it does not have any persistent volumes claims.